src/injected/visualization/tabbed-item.ts

```diff
@@ -8,4 +8,11 @@ export interface TabbedItem {
     focusIndicator?: FocusIndicator;
     tabOrder: number;
     shouldRedraw?: boolean;
+    itemType?: TabbedItemType;
+}
+
+export enum TabbedItemType {
```

Comment:
On the use of enums: we try not to use enums as the below example because their underlying values are basically dependent on position in the enum type. That is to say MissingItem is 1 and ErroredItem is 2 but if we switch the ordering it changes.

So there's two options:
1. Use enums but specify the string, like so: `MissingItem = "MissingItem"`.
2. Don't use enums but use types. I think this is the preferred approach now because it is easier to reference when using them. For example, (enum) `someType === TabbedItemType.MissingItem` vs (type) `someType === "MissingItem"`.